dev/statement_serializer.h

```diff
@@ -1260,42 +1399,36 @@ namespace sqlite_orm {
 
             template<class Ctx>
             std::string operator()(const statement_type& upd, const Ctx& context) const {
-                std::stringstream ss;
-                ss << "UPDATE";
                 table_name_collector collector([&context](std::type_index ti) {
                     return context.impl.find_table_name(ti);
                 });
                 iterate_ast(upd.set.assigns, collector);
-                if(!collector.table_names.empty()) {
-                    if(collector.table_names.size() >= 1) {
-                        ss << " '" << collector.table_names.begin()->first << "' SET";
-                        std::vector<std::string> setPairs;
-                        auto leftContext = context;
-                        leftContext.skip_table_name = true;
-                        iterate_tuple(upd.set.assigns, [&context, &leftContext, &setPairs](auto& asgn) {
-                            std::stringstream sss;
-                            sss << serialize(asgn.lhs, leftContext);
-                            sss << ' ' << asgn.serialize() << ' ';
-                            sss << serialize(asgn.rhs, context);
-                            setPairs.push_back(sss.str());
-                        });
-                        auto setPairsCount = setPairs.size();
-                        for(size_t i = 0; i < setPairsCount; ++i) {
-                            ss << ' ' << setPairs[i];
-                            if(i < setPairsCount - 1) {
-                                ss << ",";
-                            }
-                        }
-                        iterate_tuple(upd.conditions, [&context, &ss](auto& v) {
-                            ss << ' ' << serialize(v, context);
-                        });
-                        return ss.str();
-                    } else {
-                        throw std::system_error{orm_error_code::no_tables_specified};
-                    }
-                } else {
+
+                if(collector.table_names.empty()) {
+                    throw std::system_error{orm_error_code::no_tables_specified};
+                }
+                if(collector.table_names.size() < 1) {
```

Comment:
intention was to avoid calling `UPDATE` with several tables

Reply:
later it changed and calling `UPDATE` with 1+ tables became available

Reply:
current intention (from [previous code](https://github.com/fnc12/sqlite_orm/blob/45818cb388d6a57f49fcf97ef72cb6377d6ce0bc/dev/statement_serializer.h#L1251)) is designed to cover cases when expression has 0 tables mentioned. Theoretically it can happen. I wrote a code to handle such cases

Reply:
I understand. So `orm_error_code::no_tables_specified` is the exception to throw then.